### PR TITLE
Fix tutorials inconsistencies

### PR DESCRIPTION
**Pull request by vairodp**
# Description

Updating tutorials by fixing inconsistencies,  bad versioning, semantics, and grammar

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/content/basic_usage.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ alongside the observation for this timestep. The reward may also be negative or
 The agent will then be trained to maximize the reward it accumulates over many timesteps.
 
 After some timesteps, the environment may enter a terminal state. For instance, the robot may have crashed, or the agent may have succeeded in completing a task. In that case, we want to reset the environment to a new initial state. The environment issues a terminated signal to the agent if it enters such a terminal state. Sometimes we also want to end the episode after a fixed number of timesteps, in this case, the environment issues a truncated signal.
-This is a new change in API (v0.26 onwards). Earlier a common done signal was issued for an episode ending via any means. This is now changed in favour of issuing two signals - terminated and truncated.
```
Comment (RedTachyon) [inline docs/content/basic_usage.md:41]: I'm pretty sure the grammar was correct in the previous version?

Comment (vairodp) [reply docs/content/basic_usage.md]: Fixed

File: docs/content/environment_creation.md [hunk 2]
```diff
@@ -146,7 +149,7 @@ to a deterministic state. It is recommended to use the random number generator `
 base class, `gymnasium.Env`. If you only use this RNG, you do not need to worry much about seeding, *but you need to remember to
 call `super().reset(seed=seed)`* to  make sure that `gymnasium.Env` correctly seeds the RNG. 
 Once this is done, we can randomly set the state of our environment. 
-In our case, we randomly choose the agent's location and the randomly sample target positions, until it does not coincide with the agent's position.
+In our case, we randomly choose the agent's location and the random sample target positions, until it does not coincide with the agent's position.
```
Comment (pseudo-rnd-thoughts) [inline docs/content/environment_creation.md:152]: I think this should still be `randomly`

File: docs/content/handling_timelimits.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # Handling Time Limits
-In using Gym environments with reinforcement learning code, a common problem observed is how time limits are incorrectly handled. The `done` signal received (in previous versions of gym < 0.26) from `env.step` indicated whether an episode has ended. However, this signal did not distinguish between whether the episode ended due to `termination` or `truncation`. 
+In using Gymnasium environments with reinforcement learning code, a common problem observed is how time limits are incorrectly handled. The `done` signal received (in previous versions of gymnasium < 0.26) from `env.step` indicated whether an episode has ended. However, this signal did not distinguish whether the episode ended due to `termination` or `truncation`.
```
Comment (pseudo-rnd-thoughts) [inline docs/content/handling_timelimits.md:2]: "(in previous versions of OpenAI Gym < 0.26)"

File: docs/content/vectorising.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ title: Vectorising your environments
 Gymnasium provides two types of vectorized environments:
 
 - `gymnasium.vector.SyncVectorEnv`, where the different copies of the environment are executed sequentially.
```
Comment (pseudo-rnd-thoughts) [inline docs/content/vectorising.md:13]: With the new line below, should not this not be removed

File: docs/content/vectorising.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ title: Vectorising your environments
...
-- `gymnasium.vector.AsyncVectorEnv`, where the the different copies of the environment are executed in parallel using [multiprocessing](https://docs.python.org/3/library/multiprocessing.html). This creates one process per copy.
+- `gymnasium.vector.AsyncVectorEnv, where the different copies of the environment are executed in parallel using [multiprocessing](https://docs.python.org/3/library/multiprocessing.html). This creates one process per copy.
```
Comment (pseudo-rnd-thoughts) [inline docs/content/vectorising.md:14]: You are missing a ` at the end of AsyncVectorEnv

File: docs/content/environment_creation.md (hunk 2 above)
Comment (vairodp) [reply docs/content/environment_creation.md]: Fixed

File: docs/content/handling_timelimits.md (hunk 3 above)
Comment (vairodp) [reply docs/content/handling_timelimits.md]: Fixed

File: docs/content/vectorising.md (hunk 4 above)
Comment (vairodp) [reply docs/content/vectorising.md]: Fixed

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**